### ISSUE TITLE
Accessibility: MAS40B: Inspect: Name property is empty for "Image gallery" under "Visual Studio Code in Action" header

### ISSUE THREAD

**Issue by wiproaccess** [wiproaccess/vscode-website#34]
#A11yMAS
#MAS40B
#Wipro
#vscode-website

**Testing Environment:**
OS-RS1
OS Name Microsoft -Windows 10 Pro
Version-10.0.14393 Build 14393.607
Browser-Edge
Edge Version-Microsoft Edge 38.14393.607.0

**Repro Steps:**
1. Open https://code.visualstudio.com/docs in EDGE.
2. Navigate to the moving image gallery available under "Visual Studio Code in Action" header
3. Observe the "Name" property in Inspect

**Actual Result:**
The "Name" property is empty
**_Narrator Behaviour:_**
Narrator announces as "Link"

**Expected Result:**
Name property should indicate the purpose of images. So, Name property should be meaningful 

**User Impact:**
Visually impaired users  will not get to know the purpose of the link as the screen reader will not announce anything when the user arrives at the image gallery.
Providing the name property enables the compatibility with the Screen Reader, which may be used by people with disabilites.

**Screenshots:**
![name property is empty for the image gallery](https://cloud.githubusercontent.com/assets/24825441/21709878/8cd00e00-d40a-11e6-829c-740d644907fe.png)

**Comment by waderyan** [comment]
Fixed
